nixos/modules/services/mail/dovecot.nix

```diff
@@ -7,7 +7,90 @@ let
   dovecotPkg = pkgs.dovecot;
 
   baseDir = "/run/dovecot2";
-  stateDir = "/var/lib/dovecot";
+
+  sieveToPath = text:
+    if isString text && builtins.substring 0 2 text == "/*"
```

Comment:
The  `lib.hasPrefix` would be more explicit.

Reply:
done.